task_opts.go

```diff
@@ -114,6 +114,21 @@ func WithRestoreImagePath(path string) NewTaskOpts {
 	}
 }
 
+// WithRestoreWorkPath sets criu work path for create option
+func WithRestoreWorkPath(path string) NewTaskOpts {
+	return func(ctx context.Context, c *Client, ti *TaskInfo) error {
+		if ti.Options == nil {
+			ti.Options = &options.Options{}
+		}
+		opts, ok := ti.Options.(*options.Options)
+		if !ok {
+			return errors.New("invalid v2 shim create options format")
```

Comment:
suggest to use runtime v2 instead of v2 shim.

Reply:
done